### ISSUE TITLE
verdi quicksetup with non default sql server port

### ISSUE THREAD

**Issue by rikigigi** [aiidateam/aiida-core#3879]
Hi,
in aiida 1.1.1 installed from pip, following https://aiida-core.readthedocs.io/en/latest/install/installation.html , when I run
`verdi quicksetup`
and I specify a non default port, where my postgresql server is running, I get

```
postgres host [localhost]:
postgres port [5432]: 5532
postgres super user [postgres]: pippo
database [template1]:

Trying to access postgres ...
Info: database pippo_pippo_e0dd3f2f29ab1c1d2b829c504752073f already exists!
Use it (make sure it is not used by another profile)? [y/N]: y
Success: created new profile `pippo`.
Info: migrating the database.
Critical: database migration failed, probably because connection details are incorrect:
could not connect to server: Connection refused
        Is the server running on host "localhost" (::1) and accepting
        TCP/IP connections on port 5432?
could not connect to server: Connection refused
        Is the server running on host "localhost" (127.0.0.1) and accepting
        TCP/IP connections on port 5432?
```
If I use the default port everything works well

**Comment by giovannipizzi** [comment]
Thanks for the report!
@ramirezfranciscof could you try to reproduce this?
Mentioning also @ltalirz for comment

**Comment by giovannipizzi** [comment]
@ramirezfranciscof also this to be included in the labels "for next release", to be discussed?

**Comment by ltalirz** [comment]
Thanks for reporting @rikigigi 

Indeed - when a non-standard port was entered by the user during automatic postgresql detection, this entered port was not forwarded to the `verdi setup` command, which does the migration.
This issue has been fixed in #3892 , which has already been merged in to `develop`.
Feel free to give it a try (you'll need to `pip install -e .` again).
